cmd/root/root.go

```diff
@@ -97,7 +98,28 @@ func flagErrorFunc(c *cobra.Command, err error) error {
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute(ctx context.Context, cmd *cobra.Command) error {
-	// TODO: deferred panic recovery
+	defer func() {
+		r := recover()
+
+		// No panic. Return normally.
+		if r == nil {
+			return
+		}
+
+		version := build.GetInfo().Version
```

Comment:
This can panic as well. Perhaps best avoided here?

Reply:
It's very unlikely for this to panic, since `build.GetInfo()` does not return a pointer and is also covered by unit tests.

I think that risk is acceptable here.

Reply:
Added an acceptance test.